.travis.yml

```diff
@@ -24,6 +24,9 @@ install:
     - pip install -r requirements.txt
     - pip install -r test_requirements.txt --upgrade
     - pip install -r docs_requirements.txt
+# Install a new version of Sphinx that correctly handles forward refs once 1.7.0 is out that should
+# be used instead
+    - pip install git+https://github.com/jenshnielsen/sphinx.git@working_forward_ref --upgrade
```

Comment:
Nice!

Reply:
No it's not great. The alternative is to wait for 1.7.0 to be released